helpers/deploymentConfig.ts

```diff
@@ -115,6 +115,8 @@ export const ADDRESSES: PreconfiguredAddresses = {
     PTweETH_26DEC2024: "0x56107201d3e4b7Db92dEa0Edb9e0454346AEb8B5",
     FRAX: "0x10630d59848547c9F59538E2d8963D63B912C075",
     sfrxETH: "0x14AECeEc177085fd09EA07348B4E1F7Fcc030fA1",
+    rsETH: "0xfA0614E5C803E15070d31f7C38d2d430EBe68E47",
```

Comment:
Wouldn't it be better to add these addresses only to the deployment scripts that use them?